pkg/apis/types/types.go

```diff
@@ -121,4 +121,10 @@ const (
 	// OpenEBSCStorExistingPoolName is the name of the cstor pool already present on
 	// the disk that needs to be imported and renamed
 	OpenEBSCStorExistingPoolName = "import.cspi.cstor.openebs.io/existing-pool-name"
+
+	// OpenEBSCStorAllowedBDTagKey is the annotation key present that decides whether
+	// a particular BD with a tag is allowed in storage provisioning or not.
+	// This annotation can be used on SPC or CSPC to allow a particular BD(s) with tag
+	// for provisioning
+	OpenEBSCStorAllowedBDTagKey = "cstor.openebs.io/allowed-bd-tag"
```

Comment:
Is it `cstor.openebs.io/allowed-bd-tag` or `cstor.openebs.io/allowed-bd-tags`. [Here](https://github.com/openebs/cstor-operators/pull/172/files#diff-a87202891189d3ceb93dd55d219e1866R37) it is mentioned as tags